knowledge-base/grid-rows-text-ellipsis.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Prevent the Grid from wrapping text in multiline
+description: How to prevent the Grid from wrapping text in multiline
+type: how-to
+page_title: Prevent the Grid from wrapping text in multiline
+slug: grid-kb-rows-text-ellipsis
+position: 
+tags: 
+ticketid: 1499434
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am using the TelerikGrid component for Blazor. One of my columns contains a long text, like a description and the Grid wraps it in multiple lines. I would like to prevent that from happening and show the maximum amount of text depending on the column width and after that put an ellipsis (...). 
+
+## Solution
+
+In order to prevent the Grid from wrapping the text in multiple lines you can use CSS and target the `<td>` HTML tags, which contain the data. In order to make that easier you can take advantage of the [OnCellRender event]({%slug grid-column-events%}#oncellrender) that the component exposes. The solution below showcases a sample implementation.
+
```

Comment:
Consider adding a paragraph that you could also do this through templates, just in case someone already has templates